### PR TITLE
first half of the project

### PR DESCRIPTION
**Pull request by ixboy**
Hello TSE, I have decided to submit this project for review knowing that is incomplete because I need some feedback on if I am doing it correctly or not so far, also I needed to submit it for review in order to keep my weekly progress high I haven't submitted anything this whole week and the past week. I will work more on the project during the weekend and complete the whole project by the next week.

Thank You.
Enjoy your weekend.

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Good work so far.  💻 👀 ✨
- You're on the right track! Now let's try to focus on completing this project.
- Don't worry, your project structure is alright so far. You've created a method to add books to the array, which is precisely what is expected. Good work!

## However, there are some changes required 👀 🧰 🔨 🔧
- Please make sure to add a professional README file for the next review.
- Complete your project by adding all the necessary requirements.
- The next step is to display the array on your page using the DOM. Please add this feature next.
- Also, add a way to edit and delete the books on the list.

### Submit these changes and request a new code review! 😄 👍

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier)🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/* .form {
+  text-align: center;
+} */
```
Comment (bellom) [inline style.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

I trust you can remove this before merging.

**Review by bellom — APPROVED** [review]
## Hi @ixboy  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

File: style.css (hunk 1 above)
Comment (ixboy) [reply style.css]: Thank You!

Yes, I will do that now, before merging.

**Pull request merged**